src/Resources/public/administration/css/plugin-twitter.css

```diff
@@ -0,0 +1 @@
+.sw-cms-el-twitter-timeline{height:100%;width:100%;min-height:315px}.sw-cms-el-preview-twitter-timeline{position:relative}.sw-cms-el-preview-twitter-timeline img{margin:0 auto;display:block;max-height:110px}.sw-cms-el-twitter-buttons{height:100%;width:100%;min-height:315px}.sw-cms-el-preview-twitter-buttons{position:relative}.sw-cms-el-preview-twitter-buttons img{margin:0 auto;display:block;max-height:110px}
```

Comment:
should we push this file? also the minified js file below?

Reply:
no it should not. will be removed in the final version